CT-1301 Update NUMBER(38,0) (#589)
keboola/connection-docs: storage/tables/data-types/index.md

A component may provide information about column data types in its data manifest. Database data source connectors and transformations matching the storage backend (e.g., Snowflake SQL transformation on the Snowflake storage backend) will create storage tables with the same types. The database data source connectors and transformations that do NOT match the backend will create storage tables using [base types](#base-types). 

> _**Note:** When a table is created from base types, it defaults to the lengths and precisions of the target backend. For instance, in Snowflake, the NUMBER base type is created as NUMBER(38,9), which may be unexpected if the source database column is NUMBER(10,2)._  

Reviewer: ```suggestion
_**Note:** When a table is created from base types, it defaults to the lengths and precisions specific for each Storage backend. For instance, in Snowflake, the NUMBER base type is created as NUMBER(38,9), which may be unexpected if the source database column is NUMBER(10,2)._  

```

Reviewer: puvodni veta rikala "vezme se default podle backendu", to neni ted pravda - default nastavujeme my

Author: asi može byť
